sync.pl

```diff
@@ -1,136 +1,301 @@
-#!/usr/bin/perl
+#!/usr/bin/env perl
+# sync.pl by azatoth (2011), update by amorymeltzer (2019)
 
-use v5.10;
```

Comment:
`use v5.10;` is required to ensure the perl6-like `say` is available, but that's been removed